### PR TITLE
Add default templates

### PR DESCRIPTION
**Pull request by blossommojekwu**
Accept input templates into Stardoc rule

### CODE REVIEW

**Comment by c-parsons** [comment]
Please leave a comment here indicating when the PR is ready (I see you are still adding files) :)

**Comment by blossommojekwu** [comment]
PR is ready

**Comment by blossommojekwu** [comment]
PR passes and is ready

File: stardoc/stardoc.bzl [hunk 1]
```diff
@@ -152,5 +156,25 @@ non-default semantic flags required to use the given Starlark symbols.
             cfg = "host",
             executable = True,
         ),
+        "header_template": attr.label(
+            doc = "The input file template for header generated in documentation.",
+            allow_single_file = [".vm", ".txt"],
```
Comment (c-parsons) [inline stardoc/stardoc.bzl]: Whoops, we didn't resolve this discussion offline -- if you can truly only find that .vm templates are used by velocity engine, maybe lets restrict this just to ".vm". (and for the other template attributes)

**Pull request merged**